lib/Dialect/Torch/Transforms/GlobalizeObjectGraph.cpp

```diff
@@ -142,11 +143,36 @@ class ObjectGraphInfo {
       if (slotIt == nameToSlot.end())
         op->emitError() << "Reference to non-existing module slot " << slotName
                         << "in " << moduleType.getClassName();
-      usedSlots.insert(slotIt->getValue());
+      slotUses[slotIt->getValue()].insert(op);
     });
     return success();
   }
 
+  LogicalResult checkUsesForValueSemantics(Value root) {
```

Comment:
I would call this `bool isSlotInitializerUsedOnlyWithValueSemantics(Value initializer)`